engine/runtime-integration-tests/src/test/scala/org/enso/compiler/test/pass/analyse/DataflowAnalysisTest.scala

```diff
@@ -356,31 +356,31 @@ class DataflowAnalysisTest extends CompilerTest {
     val plusSymbol       = mkDynamicDep("+")
 
     // The Identifiers
-    val methodId           = mkStaticDep(method.getId())
```

Comment:
Please leave those with brackets. `getId` may involve side-effects (as in, actually inferring the new `id`) and as such should require brackets.